R/tm_g_events_term_id.R

```diff
@@ -333,7 +331,7 @@ srv_g_events_term_id <- function(id,
         )
       )
 
-      validate(need(nrow(q1[["ANL"]]) > 10, "ANL needs at least 10 data points"))
+      teal::validate_has_data(q1[["ANL"]], min_nrow = 10, msg = "ANL needs at least 10 data points")
```

Comment:
End user probably don't know what ANL is so I would change this to "Analysis dataset needs"